### PR TITLE
Add CLI util for merging multiple DICOMs of the same series into a single DICOM

### PR DESCRIPTION
**Pull request by timcogan**

### CODE REVIEW

File: pdm.lock
Comment (TidalPaladin) [inline pdm.lock:1]: Is this PR needed for our upcoming submission? If so (and if this change updates any dependency versions for the triage product) we will need to update the version table in the SDD.

File: dicom_utils/merge_series.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+from typing import List
+
+import pydicom
+from pydicom import Dataset
+from pydicom.errors import InvalidDicomError
+from pydicom.uid import generate_uid
+
+
+def merge_datasets(datasets: List[Dataset]) -> Dataset:
+    new_dataset = copy.deepcopy(datasets[0])
```
Comment (TidalPaladin) [inline dicom_utils/merge_series.py:14]: Can this be a shallow copy (for speed)?

**Review by TidalPaladin — COMMENTED** [review]
Looks good, just a few questions

File: dicom_utils/merge_series.py (hunk 1 above)
Comment (timcogan) [reply dicom_utils/merge_series.py:14]: That's probably a good idea.

**Review by TidalPaladin — APPROVED** [review]

**Pull request merged**